pdfmaker/kassia.py

```diff
@@ -77,18 +79,24 @@ def createPDF(self):
                 c.setFillColor(title_attrib['color'])
                 
                 vert_pos -= title_attrib['font_size'] + 10
-                c.setFont('EZOmega',title_attrib['font_size'])
+                c.setFont(title_attrib['font'],title_attrib['font_size'])
                 c.drawCentredString(self.paperSize[0]/2,vert_pos,title_text)
                 vert_pos -= 36
             c.setFillColor(colors.black)
 
-            neumesText = " ".join(troparion.find('neumes').text.strip().split())
-            lyricsText = " ".join(troparion.find('lyrics').text.strip().split())
+            # Get attributes for neumes
+            neume_elem = troparion.find('neumes')
+            if (neume_elem is not None):
+                neumesText = " ".join(neume_elem.text.strip().split())
+                neume_attrib = neume_elem.attrib
+                neume_attrib = self.fill_title_dict(neume_attrib)
```

Comment:
Yeah, we should definitely allow for no lyrics. I'll add in another pull request.

I added a separate dictionary filling function for lyrics and neumes so they aren't forced into any defaults from the title.